### PR TITLE
test: improve test coverage

### LINKED ISSUE #40: [REQ] improve unit test coverage
### What kind of request is this?

Other

### What is your request or suggestion?

Missing coverage in most of the packages

```
?       github.com/project-copacetic/copacetic/pkg/buildkit     [no test files]
?       github.com/project-copacetic/copacetic/pkg/patch        [no test files]
?       github.com/project-copacetic/copacetic/pkg/pkgmgr       [no test files]
?       github.com/project-copacetic/copacetic/pkg/report       [no test files]
?       github.com/project-copacetic/copacetic/pkg/types        [no test files]
ok      github.com/project-copacetic/copacetic/pkg/utils        0.003s
```

### PR DESCRIPTION
**Pull request by sozercan**
add unit tests

Related #40

### CODE REVIEW

File: pkg/patch/cmd_test.go [hunk 1]
```diff
@@ -0,0 +1,37 @@
+package patch
```
Comment (CodeMonkeyLeet) [inline pkg/patch/cmd_test.go:1]: Could you take a pass through all of the new files and make sure that the licensing header is added?

File: pkg/utils/utils.go [hunk 2]
```diff
@@ -36,3 +37,15 @@ func IsNonEmptyFile(dir, file string) bool {
 	}
 	return !info.IsDir() && info.Size() > 0
 }
+
+func CreateTempFileWithContent(dir string, dbType string) {
```
Comment (CodeMonkeyLeet) [inline pkg/utils/utils.go:41]: Consider breaking this out into a separate `pkg/test_utils` library; This function doesn't look like it's intended to be used in production given the baked in assumptions and it's good hygiene to separate out functions with these assumptions.

Also, the latest change seems to have gotten out of sync with dpkg_test.go expectations for the function.

File: pkg/patch/cmd_test.go (hunk 1 above)
Comment (sozercan) [reply pkg/patch/cmd_test.go:6]: added

File: pkg/utils/utils.go (hunk 2 above)
Comment (sozercan) [reply pkg/utils/utils.go]: good idea! moved

**Review by CodeMonkeyLeet — APPROVED** [review]
LGTM!

**Pull request merged**